docs/release/release_0_3_5.md

```diff
@@ -0,0 +1,59 @@
+# napari 0.3.5
+
+We're happy to announce the release of napari 0.3.5!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+
+## New Features
+- Allow using of custom color dictionary in labels layer (#1339) and (#1362)
+- Add performance monitoring widget (#1262)
+
+
+## Improvements
+- Factor out ImageSlice and ImageView from Image (#1343)
+
+
+## Bug Fixes
+- Fix warning for python 3.8 (#1335)
+- Fix ranger slide position (#1344)
+- Fix press down linux key press hold (#1350)
+- Fix points select all (#1358)
```

Comment:
So does this... Maybe we need to do a better job of monitoring the names of PRs...

```suggestion
- Fix crash when selecting all points (#1358)
```

?